spec/cypress/integration/admin/administrators/create-administrator.js

```diff
@@ -10,15 +10,15 @@ describe("Admin user creating another admin user", () => {
     const email = "j.smith@example.com";
 
     cy.visit(basePath);
-    cy.get("[data-test=create-admin-button").click();
+    cy.get("[data-test=create-admin-button]").click();
 
     cy.location("pathname").should("equal", `${basePath}/new`);
     cy.chooseNameAndEmailForUser(fullName, email);
 
     cy.location("pathname").should("equal", `${basePath}/new/confirm`);
     cy.get("main").should("contain", fullName);
     cy.get("main").should("contain", email);
-    cy.get("input.govuk-button").contains("Create administrator user").click();
+    cy.clickCommitButton();
```

Comment:
I think having helper methods like this is better and more readable - we could extend that in future PR's

Reply:
It will all be cucumber

Reply:
Stay tuned this weekend I want to spend some time on cypress , cucumber and page object pattern